cwms-data-api/src/test/java/cwms/cda/api/location/kind/GateChangeScaleTest.java

```diff
@@ -0,0 +1,249 @@
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package cwms.cda.api.location.kind;
+
+import com.google.common.flogger.FluentLogger;
+import cwms.cda.api.Controllers;
+import cwms.cda.api.enums.UnitSystem;
+import cwms.cda.api.errors.NotFoundException;
+import cwms.cda.data.dao.LocationsDaoImpl;
+import cwms.cda.data.dao.location.kind.BaseOutletDaoIT;
+import cwms.cda.data.dao.location.kind.OutletDao;
+import cwms.cda.data.dao.location.kind.ProjectStructureIT;
+import cwms.cda.data.dto.CwmsId;
+import cwms.cda.data.dto.Location;
+import cwms.cda.data.dto.LookupType;
+import cwms.cda.data.dto.location.kind.GateChange;
+import cwms.cda.data.dto.location.kind.GateSetting;
+import cwms.cda.data.dto.location.kind.Outlet;
+import cwms.cda.formatters.Formats;
+import fixtures.CwmsDataApiSetupCallback;
+import io.restassured.filter.log.LogDetail;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.time.Duration;
+import java.time.Instant;
+import java.time.Period;
+import java.time.ZonedDateTime;
+import java.time.temporal.ChronoUnit;
+import java.time.temporal.TemporalAmount;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import javax.servlet.http.HttpServletResponse;
+import mil.army.usace.hec.metadata.constants.NumericalConstants;
+import mil.army.usace.hec.test.database.CwmsDatabaseContainer;
+import org.jooq.DSLContext;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import usace.cwms.db.jooq.codegen.packages.CWMS_PROJECT_PACKAGE;
+import static cwms.cda.api.Controllers.*;
+import static cwms.cda.data.dao.DaoTest.getDslContext;
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.*;
+
+class GateChangeScaleTest extends BaseOutletDaoIT {
```

Comment:
Disable the test, so it's not run in automation

Reply:
Don't disable the test. use `@EnableIfProperty` (that might not be exact) so we don't need to change code to run the test when needed.

Reply:
Oh, that's a good idea, thank you.  Didn't know that was a thing.

Reply:
FYI, I'd use Property over environment variable. I don't think we have the gradle build pass in the environment but we do pass in properties. There's a few other tests that I believe use this mechanism you can look at for an example.